app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <string name="app_name">Interesting photos</string>
+    <string name="image_preview_title"></string>
```

Comment:
Unused since the title of ImagePreview activity is set explicitly with setTilte() method